docs/ref/contrib/admin/index.txt

```diff
@@ -1099,8 +1103,6 @@ subclass::
 
             # short_description functions like a model field's verbose_name
             address_report.short_description = "Address"
-            # in this example, we have used HTML tags in the output
```

Comment:
It looks like the snippet above here with `I can't determine this address.` in it needs `mark_safe()`